src/styles/blocks/catalog.scss

```diff
@@ -18,8 +18,15 @@
     border-radius: 5px;
     margin-bottom: 48px;
 
+    transition: all 0.3s ease;
```

Comment:
list all transition properties, it is better for performance. and move 0.3s to a variable